commands/music.py

```diff
@@ -308,7 +314,7 @@ async def _summon(self, ctx: commands.Context, *, channel: discord.VoiceChannel
 
         ctx.voice_state.voice = await destination.connect()
 
-    @commands.command(name='leave', aliases=['disconnect'])
+    @commands.command(name='leave', aliases=['disconnect', 'quit'])
     @commands.has_permissions(manage_guild=True)
```

Comment:
```suggestion
    @commands.has_permissions(move_members=True)
```
Manage guild seems a bit excessive to me and probably won't line up much with the final permission structure. Move members gives the disconnect ability IIRC, so I feel like this would make sense for now